pkg/operator/ceph/cluster/osd/provision_spec.go

```diff
@@ -155,6 +155,7 @@ func (c *Cluster) provisionPodTemplateSpec(osdProps osdProperties, restart v1.Re
 	}
 
 	cephv1.GetOSDAnnotations(c.spec.Annotations).ApplyToObjectMeta(&podMeta)
+	cephv1.GetOSDLabels(c.spec.Labels).ApplyToObjectMeta(&podMeta)
```

Comment:
Instead of OSD labels, we should have a separate category for `prepareosd` labels

Reply:
Sure, sounds good.